docs/data/rpc/admin-guide/development.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Development
+sidebar_position: 25
+---
+
+For local development, we recommend [downloading](https://hub.docker.com/r/stellar/quickstart) and running a local instance via [Docker Quickstart](https://github.com/stellar/quickstart) and running a local network or communicating with a live development [Testnet].
+
+:::caution
+
+We don't recommend running the Quickstart image in production. See the [deploy your own RPC instance](#deploy-your-own-rpc-instance) section.
+
+:::
+
+The Quickstart image with the RPC service can run on a standard laptop with 8GB of RAM and has minimal storage and CPU requirements. It is a single container that runs everything you need to test against a fully featured network. It contains:
+
+- [Stellar Core] - Node software that runs the network, coordinates consensus, and finalizes ledgers.
+- [Stellar RPC] server - JSON RPC server for interacting with Soroban contracts.
+- [Horizon server] - HTTP API for accessing ledger state and historical transactions.
+- [Friendbot server] - HTTP API for creating and funding new accounts on test networks.
+
+[Stellar Core]: https://github.com/stellar/stellar-core
+[Stellar RPC]: https://github.com/stellar/stellar-rpc/tree/main
+[Horizon server]: https://github.com/stellar/go/tree/master/services/horizon
+[Friendbot server]: https://github.com/stellar/go/tree/master/services/friendbot
+
+### Standalone
+
+To run a local standalone network with the Stellar Quickstart Docker image, run the following command:
+
+```bash
+docker run --rm -it \
+    -p 8000:8000 \
+    --name stellar \
+    stellar/quickstart:testing \
+    --local \
+    --enable-soroban-rpc
```

Comment:
This is not really related to the docs, but can you make an issue to track changing this flag in quickstart to `--enable-stellar-rpc`

Reply:
That's already a thing - this line can just be updated :+1: